Vagrantfile

```diff
@@ -2,8 +2,8 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/25/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-25-1.3.x86_64.vagrant-libvirt.box"
-  config.vm.box = "f25-cloud-libvirt"
+  config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-27-1.6.x86_64.vagrant-libvirt.box"
+  config.vm.box = "f27-cloud-libvirt"
```

Comment:
Can we use f28 ?

Reply:
No. It ships with new `crank` which removes some functions.
```
     File "/usr/lib/python2.7/site-packages/tg/__init__.py", line 52, in <module>
        from tg.controllers import TGController, RestController, redirect, url, lurl, abort
      File "/usr/lib/python2.7/site-packages/tg/controllers/__init__.py", line 2, in <module>
        from tg.controllers.decoratedcontroller import DecoratedController
      File "/usr/lib/python2.7/site-packages/tg/controllers/decoratedcontroller.py", line 14, in <module>
        from crank.util import get_params_with_argspec, flatten_arguments
    ImportError: cannot import name flatten_arguments
```

Reply:
Is it not possible to fix the issue and move to f28?

Reply:
No, there is no `flatten` in `fedora-packages`, so must be Turbogears. How could Fedora ship with Turbogears that doesn't work. Are there any automated sanity tests, or it is all manual maintainers burden?

Reply:
We are starting to add [CI](https://fedoraproject.org/wiki/CI) for packages, but yes currently it is all maintainers burden. 
And I only maintain Turbogears because of this app and this app is currently running on f27 :)

Reply:
Ohh, I'd like to work on that testing infrastructure. Very intriguing, but no in my "free" time, which is still about $150/week of expenses. :D